static/app/components/events/eventReplay/replayContent.tsx

```diff
@@ -12,11 +12,12 @@ import useReplayData from 'sentry/utils/replays/hooks/useReplayData';
 import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
 
 type Props = {
+  getInitialTimeOffset: (startTimestampMs: number) => number;
   orgSlug: string;
   replaySlug: string;
 };
 
-function ReplayContent({orgSlug, replaySlug}: Props) {
+function ReplayContent({orgSlug, replaySlug, getInitialTimeOffset}: Props) {
```

Comment:
What if we passed in the `event` instead of this function, and then called `relativeTimeInMs` directly?